templates/federated_reporting/pull_dumps_from.sh

```diff
@@ -32,21 +32,21 @@ feeder="$1"
 
 mkdir -p "$feeder"
 
-"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder" "test -e $CFE_FR_TRANSPORT_DIR/*.sql.$CFE_FR_COMPRESSOR_EXT" ||
+"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@{$feeder}" "test -e $CFE_FR_TRANSPORT_DIR/*.sql.$CFE_FR_COMPRESSOR_EXT" ||
   {
     log "No files to transport."
     exit 0
   }
 
 # move the files so that they don't get overwritten/deleted during the transport
-"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder" "mkdir $CFE_FR_TRANSPORT_DIR/$$.transporting"
-"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder" "mv $CFE_FR_TRANSPORT_DIR/*.sql.$CFE_FR_COMPRESSOR_EXT $CFE_FR_TRANSPORT_DIR/$$.transporting/"
+"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@{$feeder}" "mkdir $CFE_FR_TRANSPORT_DIR/$$.transporting"
+"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@{$feeder}" "mv $CFE_FR_TRANSPORT_DIR/*.sql.$CFE_FR_COMPRESSOR_EXT $CFE_FR_TRANSPORT_DIR/$$.transporting/"
 
 failed=0
-"$CFE_FR_TRANSPORTER" $CFE_FR_TRANSPORTER_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder:/$CFE_FR_TRANSPORT_DIR/$$.transporting/*.sql.$CFE_FR_COMPRESSOR_EXT" "$feeder/" ||
+"$CFE_FR_TRANSPORTER" $CFE_FR_TRANSPORTER_ARGS "$CFE_FR_FEEDER_USERNAME@{$feeder}:/$CFE_FR_TRANSPORT_DIR/$$.transporting/*.sql.$CFE_FR_COMPRESSOR_EXT" "$feeder/" ||
   failed=1
 
-"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder" "rm -rf $CFE_FR_TRANSPORT_DIR/$$.transporting"
+"$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@{$feeder}" "rm -rf $CFE_FR_TRANSPORT_DIR/$$.transporting"
```

Comment:
These changes should stay in a separate commit.

Reply:
done.